api/envoy/config/filter/http/transcoder/v2/transcoder.proto

```diff
@@ -61,4 +61,28 @@ message GrpcJsonTranscoder {
   // the match the upstream gRPC service. Note: This means that routes for gRPC services that are
   // not transcoded cannot be used in combination with *match_incoming_request_route*.
   bool match_incoming_request_route = 5;
+
+  // A list of query parameters to be ignored for transcoding method mapping.
+  // By default, the transcoder filter will not transcode a request if there are any
+  // unknown/invalid query parameters.
+  //
+  // Example :
+  //
+  // .. code-block:: proto
+  //
+  //     service Bookstore {
+  //       rpc GetShelf(GetShelfRequest) returns (Shelf) {
```

Comment:
`message Shelf` is not defined. I don't mind if it's even just an empty object i.e. without fields, but it should be defined for completeness sake.

Reply:
Added  an empty `message Shelf`.